app/code/Magento/MediaContentSynchronizationCms/Model/Synchronizer/SynchronizeIdentitiesCms.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaContentSynchronizationCms\Model\Synchronizer;
+
+use Magento\Framework\App\ResourceConnection;
+use Magento\MediaContentApi\Api\UpdateContentAssetLinksInterface;
+use Magento\MediaContentApi\Model\GetEntityContentsInterface;
+use Magento\MediaContentSynchronizationApi\Api\SynchronizeIdentitiesInterface;
+
+class SynchronizeIdentitiesCms implements SynchronizeIdentitiesInterface
```

Comment:
No need to mention module name in the class names of the module
```suggestion
class SynchronizeIdentities implements SynchronizeIdentitiesInterface
```